server/src/application/application.entity.ts

```diff
@@ -51,4 +52,7 @@ export class Application extends CustomBaseEntity {
   // change to ManyToMany accordingly if needed.
   @ManyToOne(() => User, (user) => user.applications)
   user: User;
```

Comment:
Change this to assignedTo!

Reply:
I've noticed that we already have assignedTo that holds an externalId for a user. Should that be renamed to something else, should we get rid of that column or should we get rid of the relationship all together?

Thanks in advance!

<img width="315" alt="image" src="https://user-images.githubusercontent.com/102998728/205687415-b17e559e-4f19-4235-bbe6-be6ffc116a7f.png">

Reply:
This user is the assignedTo and no table other than the User should hold the externalId of the user!